packages/mobile/src/components/CurrencyDisplay.tsx

```diff
@@ -46,6 +47,7 @@ interface Props {
   showLocalAmount?: boolean
   showExplicitPositiveSign: boolean // shows '+' for a positive amount when true (default is false)
   formatType: FormatType
+  hideFullCurrencyName: boolean
```

Comment:
There's lots of props already, I wonder if we should assign some default values to some like this one?

Reply:
check below 😄